### DISCUSSION TITLE
Ordering of lines/legends in the Fluent UI Area Chart

### DISCUSSION THREAD

**Discussion by ashwinkh-msft** [microsoft/fluentui#28098] [Q&A] [answered]
I am working on the Fluent UI charting library(area chart) to build a few visualizations. I was wondering if there is a way to sort the lines in the area chart so that the smallest value is at the bottom and the largest value is at the top, however the legends should be sorted so that the largest values shows up first(to the left) and the smaller values later?

**Comment by AtishayMsft** [comment, marked as answer]
Fluent charts do not have the capability to modify the sorting order for legends compared to the area chart currently.
We welcome contributions if you need this functionality for your scenarios.
